docs/clusters.md

```diff
@@ -276,15 +276,21 @@ Ports are configurable in the Cluster configuration.
 
 ### Retention Period
 
-A Retention Period is the amount of time a Workflow Execution Event History remains in the Cluster's persistence store.
+A Retention Period is the amount of time, in days, that a Workflow Execution Event History remains in the Cluster's persistence store.
 
-- [How to set the Retention Period for the Namespace](/tctl/namespace/register/#--retention)
+- [How to set the Retention Period for a Namespace](/tctl/namespace/register/#--retention)
+- [How to set the Retention Period for a Namespace using an SDK](/application-development/features/#namespaces)
 
 A Retention Period applies to a single [Namespace](/namespaces#) and is set when the Namespace is registered.
```

Comment:
It is weird to say that it applies to a 'single' Namespace. I would say it applies to all closed workflows within that namespace.